pages/accountLists/[accountListId]/coaches.page.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+import Head from 'next/head';
+import { useTranslation } from 'react-i18next';
+import { CoachingList } from 'src/components/Coaching/CoachingList';
+
+const CoachesPage = (): ReactElement => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Head>
+        <title>MPDX | {t('Coaches')}</title>
+      </Head>
+      <CoachingList />
+    </>
+  );
+};
+
+export default CoachesPage;
```

Comment:
Can we call this page `/coaching`? I'm open to suggestions but I think this is an opportunity to name it better.
```suggestion
export default CoachingPage;
```